docs/model_configuration.md

```diff
@@ -665,7 +665,12 @@ dynamic batcher configurations.
 The *preferred_batch_size* property indicates the batch sizes that the
 dynamic batcher should attempt to create. For most models,
 *preferred_batch_size* should not be specified, as described in
-(#recommended-configuration-process).
+(#recommended-configuration-process). An exception is TensorRT models
```

Comment:
I believe you need to add the square brackets `[Recommended Configuration Process](#recommended-configuration-process)` to self reference a section

> as described in [Recommended Configuration Process](#recommended-configuration-process)

Otherwise, LGTM